packages/ember-views/tests/views/text_field_test.js

```diff
@@ -553,7 +553,6 @@ test('should not reset cursor position when text field receives keyUp event', fu
 
   appendView(view);
 
-  view.$().val('Brosiedoon, King of the Brocean');
```

Comment:
The old logic for writing a `value` would compare the value to be set against the one currently in the DOM.

In the new logic, a cached last value is compared instead. The bound attribute does not want a 3rd party (here, a jQuery call) to change the properties.

We may need to fall back to the check-dom-every-time strategy if this is problematic.